src/utils/az.ts

```diff
@@ -254,6 +266,51 @@ export class Az implements AzApi {
       }
    }
 
+   async listManagedClustersBySubAndRG(
+      subscriptionItem: SubscriptionItem,
+      resourceGroupItem: ResourceGroupItem
+   ): Promise<Errorable<ManagedClusterItem[]>> {
+      const loginResult = await this.checkLoginAndFilters();
+      if (!loginResult.succeeded) {
+         return loginResult;
+      }
+
+      const subscriptionId = subscriptionItem.subscription.subscriptionId;
+      if (typeof subscriptionId === 'undefined') {
+         return {succeeded: false, error: 'subscriptionId undefined'};
+      }
+
+      const resourceGroup = resourceGroupItem.resourceGroup.name;
+      if (typeof resourceGroup === 'undefined') {
+         return {succeeded: false, error: 'resourceGroup undefined'};
+      }
+
+      const {credentials2, environment} = subscriptionItem.session;
```

Comment:
💡 Thoughts\question: `credentials2` is there any specific context with this and there is `credentials1` used somewhere else?

Reply:
https://github.com/Azure/aks-devx-tools/blob/e4a0b0fe06f76c1e11ae5e60fe2fad2ed78c5239/src/utils/azureAccount.api.d.ts#L41

Its required for the azure accounts